benchmarks/morphology.py

```diff
@@ -0,0 +1,22 @@
+"""Benchmarks for `skimage.morphology`."""
+
+
+import numpy as np
+from skimage import data, morphology
+
+
+class Watershed(object):
+
+    param_names = ["seed_count", "connectivity", "compactness"]
+    params = [(5, 50, 500), (1, 2), (0, 0.01)]
+
+    def setup(self, *args):
+        self.image = data.coins()
+
+    def time_watershed(self, seed_count, connectivity, compactness):
+        morphology.watershed(self.image, seed_count, connectivity,
+                             compactness=compactness)
+
+    def mem_watershed(self, seed_count, connectivity, compactness):
```

Comment:
Sorry, this is actually incorrect (my bad in the template), see the correct way to measure memory usage in #3249